crates/contract/tests/sandbox/common.rs

```diff
@@ -919,6 +915,17 @@ pub async fn assert_running_return_participants(
     Ok(running_state.parameters.participants().clone())
 }
 
+pub async fn assert_running_return_threshold(contract: &Contract) -> anyhow::Result<Threshold> {
+    let final_state: ProtocolContractState = contract.view("state").await?.json()?;
+    let ProtocolContractState::Running(running_state) = final_state else {
+        panic!(
+            "Expected contract to be in Running state: {:?}",
+            final_state
+        );
+    };
+    Ok(running_state.parameters.threshold())
+}
```

Comment:
Why have an error path for this return value? Is there any place where we would not want to panic if `view("state")` fails?